django/db/models/fields/related.py

```diff
@@ -914,7 +914,7 @@ def get_prefetch_queryset(self, instances, queryset=None):
                     False,
                     self.prefetch_cache_name)
 
-        def add(self, *objs):
+        def add(self, *objs, **kwargs):
```

Comment:
This method shouldn't have `**kwargs`—it doesn't in 1.9. The many-to-many related manager only supports bulk operations, so it isn't configurable.

I'll make a followup PR to fix this.